app/helpers/questionnaire_responses_helper.rb

```diff
@@ -0,0 +1,16 @@
+module QuestionnaireResponsesHelper
+  def latest_active_questionnaire_id(type)
+    Questionnaire.active.where(questionnaire_type: type).order(dsl_version: :desc).first.id
```

Comment:
[suggestion, non-blocking]
If there are no Questionnaires in the database, this line would throw a NoMethodError exception. Can this be a safe navigation, so that response creation fails in the `seed` method instead?
```suggestion
    Questionnaire.active.where(questionnaire_type: type).order(dsl_version: :desc).first?.id
```

If I were a downstream user of this mixin, I wouldn't conventionally expect a method called `latest_active_questionnaire_id` to throw an exception when there's no active questionnaire.

Reply:
Good suggestion. `first?` method won't work on arrays, have changed it to `first.try(:id)`